frontend/packages/kubevirt-plugin/src/k8s/requests/vm/create/common.ts

```diff
@@ -91,3 +92,18 @@ export const initializeCommonVMMetadata = (
     );
   }
 };
+
+export const initializeCommonTemplateMetadata = (
+  settings: {
+    [VMSettingsField.DESCRIPTION]: string;
+    [VMSettingsField.FLAVOR]: string;
+    [VMSettingsField.WORKLOAD_PROFILE]: string;
+    osID: string;
+    osName: string;
+  },
+  entity: VMWrapper | VMTemplateWrapper,
+  template?: TemplateKind,
+) => {
+  initializeCommonMetadata(settings, entity, template);
```

Comment:
please use the same pattern as VM and call the `initializeCommonMetadata` separately